recipes/tippecanoe/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "tippecanoe" %}
+{% set version = "1.34.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/mapbox//{{name}}/archive/{{ version }}.tar.gz
+    sha256: 7a2dd2376a93d66a82c8253a46dbfcab3eaaaaca7bf503388167b9ee251bee54
+
+build:
+  skip: true   # [win]
+  number: 0
+  script:
+    - ls
+    - make -j$CPU_COUNT
+    - make install
+
+requirements:
+  build:
+    - make
+    - {{ compiler ('c') }}
+    - {{ compiler ('cxx') }}
+  host:
+    - sqlite
+    - zlib
+
+test:
+  commands:
+    - which tippecanoe
+    - tippecanoe -- help
```

A space too much? This command is failing the build.
```suggestion
    - tippecanoe --help
```

---

thanks for catching that. it will most likely drive me crazy if thats the only issue causing it to fail.

---

@xhocky. yup. it did work